view.js

```diff
@@ -265,10 +272,22 @@ const result = dv
   .flatMap(state.showTree ? tree : (l) => (showParent(l) ? [l] : l.children))
   .filter((l) => !state.onlyWithTasks || someOfMeAndMyChildren(l, showTask))
   .groupBy((l) => (state.groupBySection ? l.link : l.link.toFile()))
-  .sort((g) => (state.sortByPath ? g : g.key.fileName()), "desc");
+  .sort((g) => (state.sortByPath ? g.key.fileName() : g), "desc")
+  ;
 
 if (state.showCopyFeedButton && state.showOptionsPanel) {
   addCopyFeedButton(result);
 }
 
-dv.taskList(result);
+if (state.hideHeaders) {
+  result = result
+    .flatMap((section) =>
+      section.rows.map((row) => ({
+        ...row,
+        text: `${section.key} - ${row.text}`,
+      }))
+    );
+  dv.taskList(result, false);
+} else {
+  dv.taskList(result);
+}
```

Comment:
Not tested, but this looks quite good

Reply:
I wasn't sure how to make that conditionally chained lol...

Reply:
i donn't think this is a bad solution!